src/main/java/org/broadinstitute/hellbender/tools/sv/SVCallRecord.java

```diff
@@ -380,7 +399,73 @@ public Double getLog10PError() {
         return log10PError;
     }
 
-    public GATKSVVCFConstants.ComplexVariantSubtype getCpxSubtype() {
+    public GATKSVVCFConstants.ComplexVariantSubtype getComplexEventSubtype() {
         return cpxSubtype;
     }
+
+    public List<ComplexEventInterval> getComplexEventIntervals() {
+        return cpxIntervals;
+    }
+
+    public static final class ComplexEventInterval implements Locatable {
+
+        private final GATKSVVCFConstants.StructuralVariantAnnotationType intervalType;
+        private final SimpleInterval interval;
+
+        public ComplexEventInterval(final String str) {
```

Comment:
This is a great point. This class is completely redundant with SVSegment, so I've moved that to its own class. However, I've made `ComplexEventInterval` a subclass with `encode()`/`decode()` methods. We could delete the new class altogether, but I think we should have this separate class since each is used in different tools. This will make it easier to make independent changes later if we need to.